src/pages/index.md

```diff
@@ -8,8 +8,8 @@ seo:
   url: "https://ironicbaremetal.org/"
 header:
   bottomtext:
-    title: 17.0.1 release available now
-    link: "https://docs.openstack.org/releasenotes/ironic/wallaby.html"
+    title: 18.0.0 release available now
+    link: "https://docs.openstack.org/releasenotes/ironic/unreleased.html"
```

Comment:
Better link to the specific release

Reply:
We don't further decompose it so this is the link which is available now.

Reply:
https://docs.openstack.org/releasenotes/ironic/unreleased.html#relnotes-18-1-0